nicegui/client.py

```diff
@@ -84,6 +85,12 @@ def is_auto_index_client(self) -> bool:
         """Return True if this client is the auto-index client."""
         return self is self.auto_index_client
 
+    @staticmethod
+    def current_client() -> Optional[Client]:
```

Comment:
Do we really need this? Why not use 

```py
from nicegui import context
context.get_client()
```

in the user code base?

Reply:
After I worked myself through all examples of NiceGUI and all (by the way really well done) document pages I never had knowingly contact with the the library **context**. De facto is the only place where it is even used in the examples the ones where you do **context.get_client()**.

On the other hand the client class is already "well known" by the users as it is used all over the place, even an optional parameter of all pages.

Or with other words: For you who is working on this project since two years things like the context lib, slots etc. might seems the obvious place to look at. For newbies they ... in my opinion ... would find something defined in the Client easier, thus the "forwarding" of this call from "high" to "low" level.

Reply:
Hmm... ok. @falkoschindler do you agree?

Reply:
Maybe it is better to add documentation on how to use the `context` object? For example instead of injecting the Client into a request in 

https://github.com/zauberzeug/nicegui/blob/bcdae282783ecabfdad11487d13e78cae223e40c/website/documentation/content/page_documentation.py#L42-L55

It is more flexible to use `context.get_client()` because this can globally be called from sub-functions.

Reply:
That client parameter injection part in the current source code definitely was one of those ***ugh*** moments when I read through the source base even though its of course just picking up the approach of passing **request** into a fastapi endpoint.

I am personally more a fan of the Flask alike, context based approach where you just do from flask import request and be done.

Removing it will though likely brick many NiceGUI apps I can imagine, would need a healthy deprecation period....

Reply:
To minimize the changes in this PR and as this is obviously still something to sleep about I removed this change for now.